lib/chef/knife/rackspace_server_create.rb

```diff
@@ -460,11 +482,35 @@ def run
         msg_pair("Environment", config[:environment] || '_default')
         msg_pair("Run List", config[:run_list].join(', '))
       end
-      
+
+      def setup_rackconnect_network!(server)
+        # I know. I know. It beats another Identity call
+        auth_token = connection.instance_variable_get("@auth_token")
```

Comment:
How about `c.authenticate`, which apparently [returns the access token](https://github.com/fog/fog/blob/master/lib/fog/rackspace/service.rb#L116) and could reauthenticate if necessary.